enterprise/server/remote_execution/runner/runner.go

```diff
@@ -212,7 +212,8 @@ func (r *taskRunner) String() string {
 		truncate(r.key.InstanceName, 8, "..."), truncate(ph, 8, ""))
 }
 
-func (r *taskRunner) pullCredentials() (oci.Credentials, error) {
+func (r *taskRunner) pullCredentials(ctx context.Context) (oci.Credentials, error) {
+	log.CtxDebugf(ctx, "Pulling credentials")
```

Comment:
fyi this doesn't pull credentials, it just returns a struct

Reply:
We should probably rename it to not be a verb-noun when it doesn't do the verb. Something like `getCredentials()` or `getPullCredentials()`, but not urgent!